test/everypolitician_test.rb

```diff
@@ -30,6 +51,21 @@ def test_legislature_find
     end
   end
 
+  def test_find_legislature_is_case_insensitive
+    VCR.use_cassette('countries_json') do
+      legislature = Everypolitician::Legislature.find('UK', 'commons')
+      assert_equal 'House of Commons', legislature.name
+    end
+  end
+
+  def test_finds_legislature_by_several_fields
+    VCR.use_cassette('countries_json') do
+      query = {lastmod: '1469382789', person_count: 475}
```

Comment:
I'm a little hesitant to even semi-document searching by these fields by enshrining them into tests. They _do_ currently work, but I'd be inclined to treat that as accidental rather than deliberate. Or, from another angle, I'm not so sure that if an implementation change were to break these, we'd care…

I'm open to persuasion that this is something we actively _want_, though…